src/dist/dist.rs

```diff
@@ -43,6 +43,14 @@ static TOOLCHAIN_CHANNELS: &[&str] = &[
     r"\d{1}\.\d{1,3}(?:\.\d{1,2})?",
 ];
 
+const TOOLSTATE_MSG: &str =
```

Comment:
The commit message mentions to `Move` this, but I don't see it being removed anywhere. Is that an oversight?

Reply:
Sorry for previously being unclear about this. I've learned a lot on how to separate a big patch into relevant small commits now.

The message was previously deleted in https://github.com/rust-lang/rustup/pull/3453/commits/9107c0235ceeb1b227e333495903b78acb135499. Maybe it's better to say that I recycled it here since this error message has been used in a wrong module for another kind of error (https://github.com/rust-lang/rustup/pull/3453#issuecomment-1685269047).

Reply:
@djc I rewrote the history to make the move actually appear in the last commit. Sorry about the inconvenience as I'm still improving my collaboration skills!